bonsai/examples/governance/methods/Cargo.toml

```diff
@@ -1,17 +1,22 @@
 [package]
-name = "bonsai-starter-methods"
+name = "methods"
```

Comment:
I think we should still prefix this, maybe `governance-methods`

Reply:
Done in cf1c802c4328f19f3d6d32961dd985c1d6acb463